test-data/full64.yaml

```diff
@@ -197,22 +194,16 @@ post:
 test-case: equals positive interval
 
 pre: ["r1.type=number", "r1.svalue=[-1, 2]",
-      "r2.type=number", "r2.svalue=[1, 3]", "r2.uvalue=[1, 3]", "r2.svalue=r2.uvalue"]
+      "r2.type=number", "r2.svalue=[1, 3]", "r2.uvalue=[1, 3]", "r2.uvalue=r2.svalue"]
 
 code:
   <start>: |
-     assume r1 == r2
+    assume r1 == r2
 
 post:
+  - r1.uvalue=r1.svalue
+  - r2.svalue=r1.svalue
+  - r2.uvalue=r1.svalue
```

Comment:
Should we list invariants in alphabetical order?  Right now the r1's aren't together in order (line 204 and 207 are separated by r2...=...)

Reply:
The test comparison is order-insensitive

Reply:
It's a good idea to make the tests more consistent though